crates/l3/starknet-instance/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+use appchain_utils::errors::Error;
+use starknet_accounts::SingleOwnerAccount;
+use starknet_ff::FieldElement;
+use starknet_providers::jsonrpc::{HttpTransport, JsonRpcClient};
+use starknet_signers::{LocalWallet, SigningKey};
+use url::Url;
+
+pub type LocalWalletSignerMiddleware =
+    SingleOwnerAccount<JsonRpcClient<HttpTransport>, LocalWallet>;
+
+#[derive(Debug)]
+pub struct StarknetClient {
+    signer: LocalWalletSignerMiddleware,
+    url: Url,
```

Comment:
I don't think this field is ever used. Is it?

Reply:
fixed